tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -812,34 +812,31 @@ ts_decompress_chunk_generate_paths(PlannerInfo *root, RelOptInfo *chunk_rel, Hyp
 			 */
 			if (batch_merge_path != NULL)
 			{
-				Path *merge_append_path =
-					(Path *) create_merge_append_path_compat(root,
-															 chunk_rel,
-															 list_make2(batch_merge_path,
-																		uncompressed_path),
-															 root->query_pathkeys,
-															 req_outer,
-															 NIL);
-
-				add_path(chunk_rel, merge_append_path);
+				path = (Path *) create_merge_append_path_compat(root,
+																chunk_rel,
+																list_make2(batch_merge_path,
+																		   uncompressed_path),
+																root->query_pathkeys,
```

Comment:
Is this really beneficial in all the cases? I would assume this is mostly beneficial when it is a child of ordered append.